src/client/eppo-client.ts

```diff
@@ -657,6 +637,25 @@ export default class EppoClient {
     return result;
   }
 
+  /**
+   * Maps a subject to a boolean variation for a given experiment.
+   *
+   * @param flagKey feature flag identifier
+   * @param subjectKey an identifier of the experiment subject, for example a user ID.
+   * @param subjectAttributes optional attributes associated with the subject, for example name and email.
+   * @param defaultValue default value to return if the subject is not part of the experiment sample
+   * @returns a boolean variation value if the subject is part of the experiment sample, otherwise the default value
+   */
+  public getBooleanAssignment(
```

Comment:
minor: what's the reason for the move of this method?

Reply:
Geniuine mistake, not sure how that happend. Reverted it back to it's original place 😅